app/models/user.js

```diff
@@ -11,7 +13,21 @@ module.exports = (sequelize, DataTypes) => {
       updatedAt: { type: DataTypes.DATE }
     },
     {
-      underscored: true
+      underscored: true,
+      hooks: {
+        beforeCreate: async user => {
+          if (user.password) {
+            const salt = await bcrypt.genSaltSync(8, 'a');
+            // eslint-disable-next-line require-atomic-updates
+            user.password = bcrypt.hashSync(user.password, salt);
+          }
+        }
+      },
+      instanceMethods: {
+        validPassword(password) {
+          return bcrypt.compare(password, this.password);
+        }
+      }
```

Comment:
The management of the encryption of the password better takes it to a separate process, this could be in the `helpers` or `utils`.